### DISCUSSION TITLE
ComfyUI Graph Navigator extension

### DISCUSSION THREAD

**Discussion by rock-land** [comfyanonymous/ComfyUI#583] [Show and tell]
I've thrown together a basic extension to help make navigating large and unwieldly graphs a bit easier. 

Try it out if you think it will be useful too:
https://github.com/rock-land/graphNavigator 

![235281310-9d909f5f-93be-4603-86ed-fc23fbf97a4f](https://user-images.githubusercontent.com/111552054/235281449-18628b2b-e5a2-41ee-a990-0b2aaec8063f.png)

**Comment by omar92** [comment]
nice one , why not merge it into comfy
